hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/InvocationMonitor.java

```diff
@@ -335,12 +338,12 @@ public void run0() {
 
             backupTimeoutsCount.inc(backupTimeouts);
             normalTimeoutsCount.inc(normalTimeouts);
-            log(invocationCount, backupTimeouts, normalTimeouts);
+            log(invocationCount, backupTimeouts, normalTimeouts, memberLeft);
         }
 
-        private void log(int invocationCount, int backupTimeouts, int invocationTimeouts) {
+        private void log(int invocationCount, int backupTimeouts, int invocationTimeouts, int memberLeft) {
             Level logLevel = null;
-            if (backupTimeouts > 0 || invocationTimeouts > 0) {
+            if (backupTimeouts > 0 || invocationTimeouts > 0 || memberLeft > 0) {
```

Comment:
Backup or invocation timeouts can be an indication of things going wrong. Is detecting a member-left situation similarly important to report at `INFO` level?

Reply:
I think it is slightly useful (and rare in production), but I can remove them if you disagree.

Reply:
To me this can be better if we log it as fine, memberLeft is mostly an expected situation.

Reply:
https://github.com/hazelcast/hazelcast/pull/23005/commits/82593a581477078d8f28c1f0ba6de8c2afe8b3f6